packages/server/graphql/mutations/resetRetroMeetingToGroupStage.ts

```diff
@@ -112,13 +112,13 @@ const resetRetroMeetingToGroupStage = {
       r.table('Task').getAll(r.args(discussionIdsToDelete), {index: 'discussionId'}).delete().run(),
       pg
         .updateTable('RetroReflectionGroup')
-        .set({voterIds: []})
+        .set({voterIds: [], summary: null, discussionPromptQuestion: null})
         .where('id', 'in', reflectionGroupIds)
         .execute(),
       r
         .table('RetroReflectionGroup')
         .getAll(r.args(reflectionGroupIds))
-        .update({voterIds: []})
+        .update({voterIds: [], summary: '', discussionPromptQuestion: ''})
```

Comment:
-1 Why don't we set the summary and question to `null` here? Seems like it's introducing an inconsistency between R and PG for no reason.